### PR TITLE
Review and update Sites

### PR DESCRIPTION
**Pull request by rachelwhitton**
## Effect
PR includes the following changes:
- Reviewed Sites page
- Fixed link in Alert
- Markdownlint compliance and formatting.
- Minor copy adjustment to align with Dashboard.

## Remaining Work
- [x] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Update Status Report (debatable on this one)
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5507 | Difference |
| --- | --- | --- | --- |
| Performance Score | `57.99999999999999` | `60` | `2` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5507` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18341/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18341/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by rachelwhitton** [comment]
@alexfornuto womp, this didn't work

File: source/content/sites.md [hunk 1]
```diff
@@ -63,48 +72,63 @@ The [Code tool](/code) on Dev/Multidev environments includes a Connection Mode t
...
 Use this tool if you need to completely wipe your database and files for a single environment. Wiping completely resets the database and files and you will lose all content for that specific environment. For example, if you wipe the Dev environment, Test and Live are not affected. You will then need to import the database and files from a backup, clone them from another environment, or re-install Drupal or WordPress for that environment.
 
 Learn more about the [Pantheon Workflow](/pantheon-workflow).
 
 ### Errors
+
 All fatal PHP errors are shown on this tab. If any are found, recommendations are made. Learn more about [PHP Errors and Exceptions](/php-errors).
+
 <Alert title="Note" type="info">
+
```
Comment (alexfornuto) [inline source/content/sites.md]: @rachelwhitton you had some whitespace on this line, which prevented it from being the empty line the MDX renderer needs to reset between Markdown and MDX. Tricky tricky.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5507 | Difference |
| --- | --- | --- | --- |
| Performance Score | `56.99999999999999` | `52` | `-5` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5507` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18349/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18349/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: source/content/sites.md (hunk 1 above)
Comment (rachelwhitton) [reply source/content/sites.md]: Ah, good to know though. Thank you!

**Pull request merged**